website/docs/d/launch_template.html.markdown

```diff
@@ -18,11 +18,25 @@ data "aws_launch_template" "default" {
 }
 ```
 
+Filter usage:
+
+```hcl
+data "aws_launch_template" "test" {
+  filter {
+    name   = "launch-template-name"
+    values = ["some-template"]
+  }
+}
+```
+
 ## Argument Reference
 
 The following arguments are supported:
 
-* `name` - (Required) The name of the launch template.
+* `name` - (Optional) The name of the launch template.
+* `filter` - (Optional) One or more name/value pairs to use as filters. There are
+several valid keys, for a full reference, check out
+[describe-launch-templates in the AWS CLI reference][1].
```

Comment:
Can you please update this documentation to the longer form configuration block style? e.g. https://github.com/terraform-providers/terraform-provider-aws/pull/12400/files#diff-259bbf21af9ff942736c706b725f4d6dR76-R88 Thank you!